apps/web/screens/Category/Category.tsx

```diff
@@ -125,6 +125,22 @@ const Category: Screen<CategoryProps> = ({
     setHash(hashString)
   }, [Router])
 
+  useEffect(() => {
+    const groupSlug = Object.keys(groups).find(
+      (x) => x === hash.replace('#', ''),
+    )
+
+    if (groupSlug) {
+      const el = itemsRef.current.find(
+        (x) => x.getAttribute('data-slug') === groupSlug,
+      )
+
+      if (el) {
+        window.scrollTo(0, el.offsetTop)
+      }
+    }
+  }, [itemsRef, groups, hash])
+
```

Comment:
Just a thought, but if you replaced `data-slug={groupSlug}` with `id={groupSlug}` [here](https://github.com/island-is/island.is/blob/02a86052c416735508578f4ccaef0563f4e782db/apps/web/screens/Category/Category.tsx#L273) would you get the same behavior without any code?

Reply:
maybe 😏 💡 😄